internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -725,11 +725,13 @@ func convertTopicSubscription(t manifest.TopicSubscription, url, accountID, app,
 	if err != nil {
 		return nil, fmt.Errorf(`invalid topic subscription "%s": %w`, aws.StringValue(t.Name), err)
 	}
-	queue, err := convertQueue(t.Queue)
+	queue, err := convertQueue(t.Queue.Advanced)
 	if err != nil {
 		return nil, fmt.Errorf(`invalid topic subscription "%s": %w`, aws.StringValue(t.Name), err)
 	}
-
+	if aws.BoolValue(t.Queue.Enabled) {
+		queue = &template.SQSQueue{}
+	}
```

Comment:
What do you think of this?
```suggestion
	if aws.BoolValue(t.Queue.Enabled) {
           return &template.TopicSubscription{
		Name:    t.Name,
		Service: t.Service,
		Queue:   &template.SQSQueue{},
	}, nil
}
if queue, err := convertQueue(t.Queue.Advanced) ...
```